### ISSUE TITLE
[HW] <NAME>nishtha arora

### ISSUE THREAD

**Issue by nishtha0608** [datastaxdevs/workshop-intro-to-cassandra#293]
**Name:** <NAME>
**Email:** <email>
**Linkedin Profile:** <LINK>

Attach the homework screenshots below for both step II and step III:
-----------------------------------------

<SCREENSHOT>

**Comment by nishtha0608** [comment]
>
![image](https://user-images.githubusercontent.com/88788270/129081104-0d48148c-cb7e-4333-b095-808b1f9d2d5d.png)

**Comment by 000** [comment]
You may need to attach "Congratulations!" screenshots from https://www.datastax.com/node/3272 and https://www.datastax.com/dev/scenario/try-it-out-cassandra-query-language-cql as well, @nishtha0608 .

**Comment by SonicDMG** [comment]
Yes, that is correct. Also, once complete please provide your name and email so we can send you your badge.
